docs/flutterinfinitelisttutorial.md

```diff
@@ -347,6 +347,7 @@ class PostBloc extends Bloc<PostEvent, PostState> {
 
   @override
   Stream<PostState> mapEventToState(event) async* {
+    final currentState = state;
```

Comment:
can we also make this change to all of the translated `flutterinfinitelisttutorial.md` files?